src/main/kotlin/ink/abb/pogo/scraper/tasks/CatchOneNearbyPokemon.kt

```diff
@@ -28,6 +28,13 @@ import ink.abb.pogo.scraper.util.pokemon.shouldTransfer
 class CatchOneNearbyPokemon : Task {
     override fun run(bot: Bot, ctx: Context, settings: Settings) {
         // STOP WALKING
+
+        if (ctx.caughtPokemon.get() > settings.pokemonThreshold) {
+            Thread.sleep(3600 * 12000)
```

Comment:
instead of sleeping, maybe set the settings.catchPokemon to false and store the time it was disabled and then re-enable it later on. See how this is done with a full pokebag: https://github.com/muschter/PokemonGoBot/blob/83c9dc6077f63f34342534076101615e0a87701a/src/main/kotlin/ink/abb/pogo/scraper/tasks/CatchOneNearbyPokemon.kt#L141